src/css/visualization.css

```diff
@@ -219,15 +219,19 @@
 }
 
 #dist-graph-container:hover .unsecured-bar {
-  background-color: rgba(33, 95, 93, 0.4);
+  background-color: rgba(117, 203, 214, 0.4);
```

Comment:
Could all this logic be simplified by doing something like:

```
#dist-graph-container:hover .dist-row:not(:hover) {
   opacity: .4;
}

Reply:
IOW if the container is hovered but the row isn’t hovered, set the rows opacity to .4. Otherwise, (if the row is hovered or the container isn’t hovered) the opacity will be 1

Reply:
Oh wow that is a lot simpler - I'll push up a follow-up PR